src/platform.ts

```diff
@@ -144,6 +145,36 @@ export class DaikinOnePlusPlatform implements DynamicPlatformPlugin {
       this.discoverOutdoorAqi(device, deviceData);
       this.discoverIndoorAqi(device, deviceData);
       this.discoverAwaySwitch(device);
+      this.discoverEmergencyHeatSwitch(device);
+    }
+  }
+
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  private discoverEmergencyHeatSwitch(device: any) {
```

Comment:
How interested are you in having proper Typescript types for things like `device` here? I didn't want to go through the work if that's not the style you intend for this plugin but might do so in other PRs if that's the case.

Reply:
I'd like to, so go ahead as you are able. I'm learning typescript as I go so I hadn't got around to figuring that part out yet.